dist/common/scripts/scylla_util.py

```diff
@@ -378,7 +378,7 @@ def disable(self):
         return run('systemctl {} disable {}'.format(self.ctlparam, self._unit), shell=True, check=True)
 
     def is_active(self):
-        return out('systemctl {} is-active {}'.format(self.ctlparam, self._unit))
+        return out('systemctl {} is-active {}'.format(self.ctlparam, self._unit), ignore_error=True)
```

Comment:
But here you are interested in the error code, not stdout.

I think we shouldn't use out() here but subprocess.run() directly.

Reply:
No, in this script we use 'active' / 'inactive' string but it actually returns non-zero returncode when 'inactive', so the script discontinue running by the exception.
Previously we just did check=false, but newly added out() function unconditonally raise exception, so this PR fixing it.

Reply:
ok. Better add documentation that it returns the string, one expects it returns bool.

Alternatively, check for active/inactive here and return bool for real.

Reply:
@avikivity Added description about "systemctl is-active" retruning string on commit log. Can we merge now?

Reply:
It should be on is_active() itself. People expect that a function named is_active() will return True/False. But if you do `if is_active(x)` it will always succeed, even if it returns "inactive".

Reply:
Okay, I will change return value to Boolean.
What about `out()` command, is it okay or maybe we should do try - catch in `is_active()` function instead of changing `out()`?

Reply:
I think a local try-catch is better.

Reply:
> Okay, I will change return value to Boolean.

No it cannot, I was forgetting `is_active()` return value is not two-state, it has more.
See why we reverted changing return value to Boolean last time: [Revert "scylla_util.py: return bool value on systemd_unit.is_active()" #9628](https://github.com/scylladb/scylla/pull/9628#).
Maybe we can rename the function but I want to keep the function name similar with original CLI if it possible, since this is just a wrapper of systemctl command.

Also, try - catch in `is_active()` was not works well,  because `out()` function currently unconditionally dump stdout and stderr when returncode is non-zero, so it will shown everytime `is_active() != "active"`.
We have to have a flag to supress the dump, so I think current patch is reasonable for the purpose (`ignore_error=True`).

Reply:
ok